terraform/fullnode/main.tf

```diff
@@ -115,6 +115,7 @@ resource "helm_release" "fullnode" {
 
   values = [
     jsonencode(var.fullnode_helm_values),
+    jsonencode(var.fullnode_helm_values_list == {} ? {} : var.fullnode_helm_values_list[count.index]),
```

Comment:
I think it makes sense to apply the helm values in order of:
* helm vaues set in this file
* var.fullnode_helm_values
* var.fullnode_helm_values_list
So they're in order of specificity 
?